Xamarin.Forms.Core/Xamarin.Forms.Core.csproj

```diff
@@ -19,7 +18,10 @@
 		<PackageReference Include="System.ComponentModel" Version="4.3.0" />
 		<PackageReference Include="System.Dynamic.Runtime" Version="4.3.0" />
 		<PackageReference Include="System.ValueTuple" Version="4.5.0" />
-	</ItemGroup>
+	</ItemGroup>	
+	<PropertyGroup>
+    <GitInfoReportImportance>high</GitInfoReportImportance>
```

Comment:
why do we report on this one specifically ?

Reply:
If we need to check for this info on the logs we only need it once.